### PR TITLE
Issue1291 - update web app integration test

### PR DESCRIPTION
**Pull request by CaroMac**

### CODE REVIEW

**Comment by CaroMac** [comment]
preview page is here https://deploy-preview-531--galasa.netlify.app/docs/running-simbank-tests/web-app-integration-test

File: src/markdown-pages/docs/managers/zos-program-manager.md [hunk 1]
```diff
@@ -3,109 +3,73 @@ path: "/docs/managers/zos-program-manager"
...
 
-</details>
+The Manager retrieves the source from the test bundle, builds and submits the relevant compile and link JCL based on  the programs attributes and CPS properties. The batch job is saved with the test run archive. The  program can be executed in the test by retrieving the library containing the load module by using  the <code>getLoadLibrary()</code> method. <br><br>  The Simbank tutorial <a href="/docs/running-simbank-tests/batch-accounts-open-test">BatchAccountsOpenTest</a> contains an example of running a simulated z/OS program called SIMBANK by using the <code>EXEC PGM=SIMBANK</code> command. <br><br> <br><br> You can view the <a href="https://javadoc.galasa.dev/index.html?overview-summary.html" target="_blank" rel="noopener noreferrer">Javadoc documentation for the Manager here</a>. <br><br>
```
Comment (hobbit1983) [inline src/markdown-pages/docs/managers/zos-program-manager.md:18]: We should remove the reference to the SIMBANK example here as that program isn't compiled by the test and isn't relevant here

File: src/markdown-pages/docs/managers/zos-program-manager.md [hunk 2]
```diff
@@ -142,15 +106,89 @@ The following are properties used to configure the zOS Program Manager.
...
+| Attribute: `language` |  The programming language. See <a href="https://javadoc.galasa.dev/dev/galasa/zosprogram/ZosProgram.Language.html" target="_blank" rel="noopener noreferrer">ZosProgram.Language</a>. <br><br>  |
+| Attribute: `cics` |  Is a CICS program and requires the CICS translator. |
+| Attribute: `loadlib` |  The load module data set name |
+| Attribute: `imageTag` |  The <code>imageTag</code> is used to identify the z/OS image. |
+| Attribute: `compile` |  Compile this zOS program. |
+| Syntax: | @ZosImage(imageTag="A")<br> public IZosImage zosImageA;<br> @ZosProgram(imageTag="A")<br> public IZosProgram zosProgramA;<br></code> |
+| Notes: | The <code>IZosProgram</code> interface has a number of methods to manage the z/OS Program. See <a href="https://javadoc.galasa.dev/dev/galasa/zosprogram/ZosProgram.html" target="_blank">ZosProgram</a> and <a href="https://javadoc.galasa.dev/dev/galasa/zosprogram/IZosProgram.html" target="_blank">IZosProgram</a> to find out more. |
```
Comment (hobbit1983) [inline src/markdown-pages/docs/managers/zos-program-manager.md:141]: We should list if these are optional or not and if they have a default value

File: src/markdown-pages/docs/managers/zos3270terminal-manager.md [hunk 3]
```diff
@@ -3,12 +3,135 @@ path: "/docs/managers/zos3270terminal-manager"
 title: "Zos3270Terminal Manager"
 ---
 
-**Alpha**
+This Manager is at Alpha level. You can view the <a href="https://javadoc.galasa.dev/index.html?overview-summary.html">Javadoc documentation for the Manager here</a>.<br>
```
Comment (hobbit1983) [inline src/markdown-pages/docs/managers/zos3270terminal-manager.md:6]: I wouldn't consider this at alpha any more - a lot of people are using it

**Review by hobbit1983 — CHANGES_REQUESTED** [review]

File: src/markdown-pages/docs/managers/zos-program-manager.md (hunk 2 above)
Comment (CaroMac) [reply src/markdown-pages/docs/managers/zos-program-manager.md]: Added

File: src/markdown-pages/docs/managers/zos-program-manager.md (hunk 1 above)
Comment (CaroMac) [reply src/markdown-pages/docs/managers/zos-program-manager.md]: Removed. Thank you.

**Review by hobbit1983 — APPROVED** [review]

**Pull request merged**